Election Create routing

cikzh opened the pull request:
Added actual navigation for the election create pages and put it in its own feature.
<!--
- What's the scope of the changes?
- What did you test? Which edge cases did you explicitly take into account?
- Are there things you want reviewers to definitely take a look at?
- Are there specific people you want feedback from?
- Do you have tips on how to test? (For example: data setup, triggering specific errors, etc.)
-->

praseodym reviewed (changes requested):
I see this on `/elections/create/check-definition` after uploading an election definition EML file:
<img width="657" alt="Screenshot 2025-05-02 at 17 46 39" src="https://github.com/user-attachments/assets/f68bea35-6b4b-4e81-9329-1ec3dccfd2cc" />

praseodym reviewed (changes requested):
Navigation works fine now!

The `NavBar` still needs some attention:
- on `/elections/create` the admin links (Verkiezingen/Gebruikers/Werkplekken/Logs) are shown
- on `/elections/create/check-definition` (the next page), no links are shown

Looking at the Figma design, the admin links need to be shown for all steps too.

jorisleker reviewed (changes requested):
Sorry @cikzh, the design was updated whilst you were building it:
- one step was added to this flow ('Aantal kiesgerechtigden').
- items after the current step are disabled (just like in data entry)
- items before the current step have the check icon 

Some other things that I noticed:
- Items in the menu that are not clickable should not have the hover effect

cikzh commented:
I'll put the PR in draft until I've implemented it with the new design

cikzh commented:
@praseodym Fixed links and also added a check for the user's role.

@jorisleker Added menu item and added hardcoded `disabled` attributes. The logic for displaying the menu items (including the checkmarks) will be implemented in another PR once we more pages than just the first.
